### PR TITLE
add dev-qt/*-6.5.1

### PR DESCRIPTION
**Pull request by Chiitoo**

### CODE REVIEW

File: dev-qt/qttools/qttools-6.5.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	assistant? ( =dev-qt/qtbase-${PV}*[sql,widgets] )
+	designer? ( =dev-qt/qtbase-${PV}*[widgets] )
+	distancefieldgenerator? (
+		=dev-qt/qtbase-${PV}*[widgets]
+		=dev-qt/qtdeclarative-${PV}*
+	)
+	pixeltool? ( =dev-qt/qtbase-${PV}*[widgets] )
+	qdbus? ( =dev-qt/qtbase-${PV}*[widgets] )
+	qdoc? ( sys-devel/clang:= )
+	qtdiag? ( =dev-qt/qtbase-${PV}*[opengl,widgets] )
```
Comment (a17r) [inline dev-qt/qttools/qttools-6.5.1.ebuild]: this would seem to be at odds with Qt proj's no `qt` prefix policy.

**Review by a17r — CHANGES_REQUESTED** [review]
<zzag[m]> If you build qt6 from source code, apply this patch https://invent.kde.org/-/snippets/2696 to qtbase

**Comment by ionenwks** [comment]
> Regarding qtmultimedia, after some chat at qt-labs, it seems to be going mainly for ffmpeg, so in theory could disable everything else but ffmpeg, unless an actual use-case for the other plug-ins arises.

Haven't looked that deep into it but, given that, personally think I would have `USE="+ffmpeg gstreamer"` *and* `REQUIRED_USE="|| ( ffmpeg gstreamer )"` to ensure there is always at least one enabled. The reason revdeps depend on `[gstreamer]` is probably because they actually needed either enabled and it'd be complicated for them to check for "at least one".

File: dev-qt/qttools/qttools-6.5.1.ebuild (hunk 1 above)
Comment (Chiitoo) [reply dev-qt/qttools/qttools-6.5.1.ebuild]: Changed for a couple of other ones too.

**Comment by Chiitoo** [comment]
> <zzag[m]> If you build qt6 from source code, apply this patch https://invent.kde.org/-/snippets/2696 to qtbase [some revdeps fail to build otherwise]

This patch doesn't apply as is hmmm will see if I can refactor it.

**Comment by Chiitoo** [comment]
> > <zzag[m]> If you build qt6 from source code, apply this patch https://invent.kde.org/-/snippets/2696 to qtbase [some revdeps fail to build otherwise]
> 
> This patch doesn't apply as is hmmm will see if I can refactor it.

Oh, it's already there:

https://code.qt.io/cgit/qt/qtbase.git/commit/?h=6.5.1&id=50d4e338cf11e83e9d8bb1e0254da60cc528a666

**Comment by Chiitoo** [comment]
Need to also fix the CVE patch.  Some parts are not in 6.5.1 that it's trying to patch, and it might be missing some of the required changes.

**Comment by Chiitoo** [comment]
Pushed to ::gentoo.

**Pull request closed without merging**